GMS.tex

```diff
@@ -26,7 +26,7 @@
 \end{abstract}
 
 \section*{Acknowledgments}
-For the creation of this document we acknowledge the support of the Canadian Space Agency, the National Research Council Canada, the Italian National Institue of Astrophysics, and the Astronomy ESFRI and Research Infrastructure Cluster (ASTERICS).  ASTERICS is a project supported by the European Commission Framework Programme Horizon 2020 Research and Innovation action under grant agreement n. 653477.
+For the creation of this document we acknowledge the support of the Canadian Space Agency, the National Research Council Canada, the Italian National Institue of Astrophysics, and the Astronomy ESFRI and Research Infrastructure Cluster (ASTERICS).  ASTERICS was a project supported by the European Commission Framework Programme Horizon 2020 Research and Innovation action under grant agreement n. 653477.
```

Comment:
what about simply:
"...Cluster (ASTERICS, European Commission Horizon 2020 RIA grant agreement n. 653477)."
It eliminates the need for "is/was".